pages/index.html

```diff
@@ -15,7 +15,7 @@ <h1>{% trans "DJ Your Way" %}</h1>
 
 {% block body %}
 <div class="gapfiller" style="margin-bottom: 0px;">
-  <p>Free and open source DJ software for Windows, macOS, and Linux</p>
+  <p><strong>Totally Free</strong> and open source DJ software for Windows, macOS, and Linux</p>
```

Comment:
I don't think we need this extra emphasis.

Reply:
done